UPDATE DOCS - FIX FOR Module '"@capacitor-community/barcode-scanner"' has no exported member 'BarcodeFormat'.

I fixed using 

`import { BarcodeScanner, SupportedFormat } from '@capacitor-community/barcode-scanner';`

and 

` const result = await BarcodeScanner.startScan({ targetedFormats: [SupportedFormat.QR_CODE] });`

Figured i would help community and submit this. It does seem to have a direct impact on performance to only be scanning for the supportedFormat wanted.

Using the documented code to target specific formats throws error on ionic cap sync. 

See below output:

error TS2322: Type '"QR_CODE"' is not assignable to type 'SupportedFormat'.

88    const result = await BarcodeScanner.startScan({ targetedFormats: ['QR_CODE'] });

And the resolution is what i originally commented.

--- a/README.md
+++ b/README.md
@@ -352,9 +352,18 @@ const checkPermission = async () => {
 
 You can setup the scanner to only recognize specific types of barcodes like this:
 
-`BarcodeScanner.startScan({ targetedFormats: ['QR_CODE'] })`
+```ts
+import { BarcodeScanner, SupportedFormat } from '@capacitor-community/barcode-scanner';
+
+BarcodeScanner.startScan({ targetedFormats: [SupportedFormat.QR_CODE] }); // this will now only target QR-codes
+```
 
 If `targetedFormats` is _not specified_ or _left empty_, _all types_ of barcodes will be targeted.
+
+Targeting only specific types can have the following benefits:
+
+- Improved performance (since the decoder only has to look for the specified barcodes)
+- Improved User Experience (since scanning a barcode that is not supported by your case, will not work)
 
 The following types are supported:
 

